Jenkinsfile

```diff
@@ -41,34 +43,60 @@ pipeline {
     WEBHOOK_ID = 'WEBHOOK_IQ'
   }
   stages {
+    stage('Hotfix') {
+      when {
+        expression { isHotfix }
+      }
+      steps {
+        script {
+          lastStage = env.STAGE_NAME
+          if (releaseTag.noCommitsOnHotfixBranch(env.BRANCH_NAME)) {
+            skip = true
+            echo "SKIP: Skipping the rest of the build because there are no commits on the hotfix branch yet"
+            return
+          }
```

Comment:
Put these statements into explicit `else` clause to avoid a one-armed `if` statement.

Reply:
Changed.